data/simple/context.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I'm conflicted on whether to include this data in the `/data/simple` folder.  It made the `eg-import-simple-election` target work, right up until I added the `--guardian-keys=./guardian_private_data.json`, but then I thought it might be a bad idea to include a bunch of guardian private keys in the source code, even if it is just for a fake election.  So now I'm thinking I should just delete it all.  Thoughts? cc @keithrfung

Reply:
The data should not be included here but PR'd and included at the electionguard folder. Anything in the election record is fine to have available. I'd actually suggest making a full election record and having that in the data. There is one that could be replaced.